README.md

```diff
@@ -27,7 +27,7 @@ Currently only Windows and Linux are supported.
 ### Installation
 Install `tricore-probe`:
 ```shell
-cargo install tricore-probe --git https://github.com/veecle/tricore-probe --version 0.2.0
+cargo install tricore-probe --git https://github.com/veecle/tricore-probe --version 0.2.1
```

Comment:
```suggestion
cargo install tricore-probe --git https://github.com/veecle/tricore-probe
```

I don't think specifying the version is useful when installing from a git repo, I didn't realize cargo would even accept it, I assume it just errors if it mismatches.